src/lib/github/updateExistingIssues.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * API function to add comments to existing bot created issues.
+ */
+
+import { Context } from "./types";
+import { Issue } from "./addIssuesToRepo";
+import { getBasicRepoProps } from "./utils";
+
+export default async function updateExistingIssues (context: Context, newIssue: Issue): Promise<void> {
+  const octokit = context.github;
+
+  const options = {
+    filter: "subscribed",
+    state: "open",
+    sort: "created",
+    order: "asc"
+  };
+
+  const results = await octokit.issues.getAll(options);
+
+  results.issues.forEach( async (issue: Issue) => {
+  if (issue.title === newIssue.title ) {
```

Comment:
the `if` block with in this `forEach` block should start with either 2 or 4 space indentation.